README.md

```diff
@@ -24,5 +24,8 @@
 | [Processing Foundation Fellowship](https://processingfoundation.org/fellowships/) | Yes |
 | [Segment Open Fellowship](https://open.segment.com/fellowship) | Yes |
 | [RARE Technologies Student Incubator Program](https://rare-technologies.com/incubator/#details) | Yes |
+| [Rosetta Commons Internship](https://www.rosettacommons.org/about/intern) | Yes** |
```

Comment:
I nowhere found it listed that it's all open source, can you please point me to it?

Reply:
I actually found it on an existing list of internship under the FOSS category https://github.com/svaksha/diversity-index/blob/master/di-internships.md

Reply:
aah, cool. We keep this out of this list. Thanks for sending a pr.